spec_parser/utils.py

```diff
@@ -405,6 +438,31 @@ def _gen_rdf(self, g: rdflib.Graph) -> None:
         g.add((cur, RDFS.comment, Literal(self.description)))
         g.add((cur, NS0.term_status, Literal(self.metadata.get("Status")[0])))
 
+    def _gen_shacl(self, g: rdflib.Graph) -> None:
+
+        class_uri = URIRef(self.metadata["id"][0])
+        shape_uri = URIRef(self.metadata["id"][0] + "Shape")
```

Comment:
There is no need to generate separate "shape" nodes. By using the same node (as the one that is a `class`), you can skip the `targetClass` property.